config/application.rb

```diff
@@ -28,6 +28,8 @@ class Application < Rails::Application
     # Load customization from special file
     config.customization = config_for(:customization)
 
+    config.autoload_paths << Rails.root.join('lib')
```

Comment:
Avoid adding `lib` to `autoload_paths` or `eager_load_paths`. It loads everything from there, including (but not limited to) rake tasks, which is not desirable.